x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -395,6 +452,22 @@ export const AddExceptionModal = memo(function AddExceptionModal({
               )}
               <EuiText>{i18n.EXCEPTION_BUILDER_INFO}</EuiText>
               <EuiSpacer />
+              {exceptionListType === 'endpoint' && !hasAlertData && (
+                <>
+                  <EuiFormRow label={i18n.OPERATING_SYSTEM_LABEL}>
+                    <EuiComboBox
```

Comment:
Just curious: was there a specific need to use the ComboBox instead of SuperSelect?

Reply:
no specific need, it just looked like the component I needed from Eui.  I'm happy to SuperSelect if it's more appropriate